### DISCUSSION TITLE
Practica de CSS duda

### DISCUSSION THREAD

**Discussion by RaulGuereca** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#311] [Q&A]
En la pagina que tenemos que clonar tego una duda tiene que ser la pagina vertical o son tres paginas o una pagina con tres divisiones, alguien me podria decir como tiene que estar estructurada la pagina a clonar.  gracias

**Comment by JonathanJRodriguez** [comment]
Hola Raul,

En mi caso lo hice como una sola página.
Si te das cuenta solo hay un pie de página, así que creo que esa es una buena referencia.

Saludos.

**Comment by doguedogue** [comment]
Es una sola página, se llaman single page website, puedes usar el tag section para separar las secciones (obvi 😂) y vas navegando por los espacios, creo que son 4 secciones, un header (navbar) y el footer.

**Reply by AngelCruzO** [reply]
De hecho es una landing page y los enlaces del navbar, son paginas independientes

**Reply by doguedogue** [reply]
Gracias, tienes razón, la barra de navegación puede mandar a otros destinos (rutas) pero que no fueron especificados en los requisitos.

Aunque creo que la duda era si eran varias páginas, yo en particular creo que es solo una.

https://es.m.wikipedia.org/wiki/P%C3%A1gina_de_aterrizaje

https://es.wikipedia.org/wiki/Single-page_application

**Reply by AngelCruzO** [reply]
Si viene especificado, como parte del bonus. Se comenta que tienes libertad de diseño para poder realizar las paginas secundarias

**Comment by JuanSalvador73** [comment]
Es una landing page, es decir, solo una. Solo que ya construída en HTML y en brower le das scroll para ver el contenido restante, hacia abajo. Y si, la idea es que tiene que se lo mas similar posible, pero el MC comentó que teníamos libertad en ciertos aspectos de diseño y funcionalidad.
